utils/plans.ts

```diff
@@ -1,5 +1,7 @@
 export function getPlanName(months: number) {
-  const { t } = useI18n();
+  const { $i18n } = useNuxtApp();
+  const t = $i18n.t;
```

Comment:
do these run on SSR? where do we call them? we should probably try to pass the `t` from outside